packages/util-address/src/index.ts

```diff
@@ -1,3 +1,7 @@
+
+export function sansPrefix(address: null): null;
+export function sansPrefix(address: string): string;
+export function sansPrefix(address: string | null): string | null;
```

Comment:
This overload is a repeat of https://github.com/onflow/fcl-js/pull/1803/files/452e6f7f5a44e5789913827be58841015707d724#diff-f08f94e9c047756c81e631cc83a0de116e474c56fa24e0c7b020679cff62b11fR10

Reply:
oh ok gotcha